README.md

```diff
@@ -9,7 +9,7 @@ Compatibility
 
 * Ember.js v3.20 or above
 * Ember CLI v3.20 or above
-* Node.js v12 or above
+* Node.js v14 or above
```

Comment:
Just want to flag up: this is breaking! That's fine as long as it's intentional. 😅

Reply:
it is!

node 12 is EOL'd, and pnpm doesn't support it